third_party/silabs/silabs_board.gni

```diff
@@ -69,6 +69,7 @@ if (silabs_board == "BRD4338A") {
   silabs_family = "SiWx917-common"
   silabs_mcu = "SiWG917M111MGTBA"
   wifi_soc = true
+  sl_uart_log_output = true
```

Comment:
This will make it so `sl_uart_log_output` cannot be overwritten to false by command line arg for 917. Is that what you want? If you want it to be configurable but default to true for 917 you have to do like what was done for 
`use_rps_extension` lower in this same file

Reply:
updated